app/packs/controllers/datatables_controller.js

```diff
@@ -2,11 +2,24 @@ import { Controller } from "stimulus"
 import { DataTable } from "simple-datatables"
 
 export default class extends Controller {
+  static values = { selector: String }
+
   connect() {
-    new DataTable("table", {
-      columns: [
-        { select: 4, sort: "desc" },  // Sort the fifth column in ascending order
-        { select: [1, 5, 6, 7, 8], sortable: false } // Disable unsortable columns
-      ]})
+    if (this.selectorValue == 'works') {
```

Comment:
Rather than having this conditional, how about splitting this into two separate controllers?  On for collections and one for works?

Reply:
good idea, done. Thanks :-)